src/Utils/decode-wa-message.ts

```diff
@@ -39,6 +39,19 @@ export function decodeMessageNode(
 			chatId = from
 		}
 
+		msgType = 'chat'
+		author = from
+	} else if(isLidUser(from)) {
```

Comment:
can we replace this else if statement with an `if(isJidUser(from) || isLidUser(from))` on line 31?

Reply:
```ts
if (isJidUser(from) || isLidUser(from)) {
  if (recipient && !isMe(from)) {
    throw new Boom('recipient present, but msg not from me', { data: stanza });
  }
  chatId = recipient || from;
  msgType = 'chat';
  author = from;
}
```